client_plugin/drivers/shared/dockerops/dockerops.go

```diff
@@ -142,13 +175,165 @@ func (d *DockerOps) VolumeRemove(volName string) error {
 }
 
 // StartSMBServer - Start SMB server
-func (d *DockerOps) StartSMBServer(volName string) bool {
-	log.Errorf("startSMBServer to be implemented")
-	return true
+func (d *DockerOps) StartSMBServer(volName string) (int, string, bool) {
```

Comment:
Can you comment what's the returned values meaning?
int: port number
string: service name
bool: success or not

Also I think it might be better to return port number as string instead of int?